conf/comps-qubes.xml

```diff
@@ -1171,12 +1171,12 @@
   </group>
 
   <group>
-    <id>debian-8</id>
-    <name>Debian 8 (jessie) template</name>
+    <id>debian</id>
+    <name>Debian 9 (stretch) template</name>
     <default>true</default>
     <uservisible>true</uservisible>
     <packagelist>
-      <packagereq>qubes-template-debian-8</packagereq>
+      <packagereq>qubes-template-stretch</packagereq>
```

Comment:
Package name should be qubes-template-debian-9. See how it is done for debian-8 in builder.conf.

Reply:
OK, I think the commit below and the one I just added to qubes-builder should address it. I want to do a full build to make sure but those are taking me ~6 hours so I'm going to incorporate a couple other tweaks first...

Reply:
Should I cancel those PRs and resubmit once I've verified?

Reply:
No, keep this one open and let me know the result.